nav2_mppi_controller/include/nav2_mppi_controller/models/constraints.hpp

```diff
@@ -41,6 +41,20 @@ struct SamplingStd
   float wz;
 };
 
+/**
+ * @struct mppi::models::AccelConstraints
+ * @brief Constraints on acceleration and deceleration
+ */
+struct AccelConstraints
+{
+  float ax_max;
+  float ax_min;
+  float ay_max;
+  float ay_min;
+  float alphaz_max;
```

Comment:
```suggestion
  float az_max;
```

There's no reason to have angular velocity min and max. Its just one value - since its rotating left/right. There's no reason a vehicle would want to accelerate differently left than right (hence you see in ControlConstraints only 1 value for `wz`  - no min/max